ghmirror/core/mirror_requests.py

```diff
@@ -98,6 +98,15 @@ def _online_request(method, url, cached_response,
         cached_response.headers['X-Cache'] = 'API_TIMEOUT_HIT'
         return cached_response
 
+    except requests.exceptions.ConnectionError:
```

Comment:
Not sure if it makes sense, but should we further distinguish between throttling issues and common connection issues? Like retry on a common connection issue that is not throttling related?